spec/unit/provider/service/launchd_spec.rb

```diff
@@ -3,6 +3,10 @@
 
 require 'spec_helper'
 
+if Puppet.features.microsoft_windows?
```

Comment:
I thought this bit wasn't necessary anymore given the change to `lib/puppet/util/plist` ?